source/settings/plugin/fts-plugin.rst

```diff
@@ -344,34 +336,50 @@ Settings
    - The asterisk can only be used at the end of the header name.
      Prefix and infix usage of asterisk are not supported.
 
-   Example:
+   Example::
 
-   .. code-block:: none
+     fts_header_excludes {
+       Received = yes
+       DKIM-* = yes
+       X-* = yes
+       Comments = yes
+     }
 
-     plugin {
-       fts_header_excludes = Received DKIM-* X-* Comments
-       fts_header_includes = X-Spam-Status Comments
+     fts_header_includes {
+       X-Spam-Status = yes
+       Comments = yes
      }
 
    - ``Received`` headers, all ``DKIM-`` headers and all ``X-`` experimental
      headers are excluded, with the following exceptions:
-
-     - ``Comments`` and ``X-Spam-Status`` are indexed anyway, as they match
-       **both** ``excludes`` and ``includes`` lists.
-     - All other headers are indexed.
+   - ``Comments`` and ``X-Spam-Status`` are indexed anyway, as they match
+     **both** ``excludes`` and ``includes`` lists and ``fts_header_includes``
+     takes precedence.
+   - All other headers are indexed.
 
    Example::
 
-     plugin {
-       fts_header_excludes = *
-       fts_header_includes = From To Cc Bcc Subject Message-ID In-* X-CustomApp-*
+     fts_header_excludes {
+       * = yes
+     }
+
+     fts_header_includes {
+       From = yes
+       To = yes
+       Cc = yes
+       Bcc = yes
+       Subject = yes
+       Message-ID = yes
+       In-* = yes
+       X-CustomApp-* = yes
      }
 
-   - No headers are indexed, except those specified in the ``includes``.
+   - No headers are indexed, except those specified in ``fts_header_includes``.
```

Comment:
Here and probably also elsewhere I hadn't noticed would be better to link to the setting.